### PR TITLE
[FIX] mail: Don't allow thread preview name to use multiple lines

### PR DESCRIPTION
**Pull request by xdu-odoo**

### CODE REVIEW

File: addons/mail/static/src/messaging/component/thread_preview/thread_preview.scss [hunk 1]
```diff
@@ -57,6 +57,7 @@
 
 .o_ThreadPreview_name {
     @include o-mail-notification-list-item-name-layout();
+    white-space: nowrap;
```
Comment (alexkuhn) [inline addons/mail/static/src/messaging/component/thread_preview/thread_preview.scss:60]: This style rule seems useless: toggling it off doesn't change anything but keep ellipsis.
So this looks fixed without this PR.
Could you confirm this style rule does nothing?

Comment (xdu-odoo) [reply addons/mail/static/src/messaging/component/thread_preview/thread_preview.scss:60]: Did you try with a long sentence like "general, this is the general channel and you should totally look at this" ? From my side, with that name, if i toggle that rule, I see the difference

Comment (alexkuhn) [reply addons/mail/static/src/messaging/component/thread_preview/thread_preview.scss:60]: Oh I see, tried a long name without whitespaces... Gonna try again but it makes sense now ^^

**Review by alexkuhn — APPROVED** [review]
ok, just reword commit message: "inline thread preview name"

**Pull request merged**